### PR TITLE
add docs for lighthouse offline audits

### PR DESCRIPTION
**Pull request by kaycebasques**
FYI failing CI test can be ignored

### CODE REVIEW

File: src/content/en/tools/lighthouse/cache-contains-start_url.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Cache contains start_url from manifest" Lighthouse audit.
+
+{# wf_updated_on: 2016-09-15 #}
+{# wf_published_on: 2016-09-15 #}
+
+# Cache contains start_url from manifest  {: .page-title }
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/cache-contains-start_url.md]: The new way is H1 is title cased. All other headings are sentence cased.

File: src/content/en/tools/lighthouse/cache-contains-start_url.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+This is a step-by-step, hands-on codelab in which you add "add to
+homescreen" functionality into an existing app. Use what you learn in
+this codelab to integrate "add to homescreen" functionality in your own app.
+
+For more help on how to cache files with service workers for offline use,
+see the "How to pass the audit" section of the following Lighthouse doc:
+[URL responds with a 200 when offline](http-200-when-offline#how)
+
+## What the audit tests for {: #what }
+
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/cache-contains-start_url.md]: This seems like it should be at the top.

File: src/content/en/tools/lighthouse/registered-service-worker.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+* Offline
+* Push notifications
+* Add to homescreen
+
+## How to pass the audit {: #how }
+
+Registering a service worker involves only a few lines of code, but the only
+reason you'd use a service worker is to implement one of the progressive
+web app features outlined above. Implementing those features require more
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/registered-service-worker.md]: require -> requires. The subject, 'implementing' is singular.

File: src/content/en/tools/lighthouse/registered-service-worker.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+work.
+
+For more help on caching files for offline use, see the "How to pass the
+audit" section of the following Lighthouse doc: [URL responds with a 200 when
+offline](http-200-when-offline#how)
```
Comment (jpmedley) [inline src/content/en/tools/lighthouse/registered-service-worker.md]: Missing period.

Comment (jpmedley) [inline src/content/en/tools/lighthouse/registered-service-worker.md]: I fixed this one already. I don't know why it's in my review.

File: src/content/en/tools/lighthouse/cache-contains-start_url.md (hunk 1 above)
Comment (kaycebasques) [inline src/content/en/tools/lighthouse/cache-contains-start_url.md]: I wrote these titles to match how they are presented in Lighthouse reports, but I guess it's not a big deal to change them so they match up with the rest of /web.

**Comment by PaulKinlan** [comment]
@petele outside of minor nits, these seems mergable.

**Comment by petele** [comment]
`lighthouse` vs `next-version`

**Comment by kaycebasques** [comment]
@petele good to go

**Pull request merged**